streaming/java/streaming-runtime/pom_template.xml

```diff
@@ -46,12 +47,33 @@
       <version>${ray.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>io.ant-ray</groupId>
+      <artifactId>common-libs-actor-optimizes</artifactId>
+      <version>0.1.5-SNAPSHOT</version>
+      <exclusions>
+        <exclusion>
+          <groupId>com.google.guava</groupId>
+          <artifactId>*</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
     <dependency>
       <groupId>io.ray</groupId>
       <artifactId>streaming-api</artifactId>
       <version>${streaming.version}</version>
     </dependency>
-    {generated_bzl_deps}
+    <dependency>
+      <groupId>io.ray</groupId>
+      <artifactId>streaming-common</artifactId>
+      <version>${streaming.version}</version>
```

Comment:
streaming-api depends on streaming-common, so streaming-runtime no need to depend on streaming-common directly